### DISCUSSION TITLE
IgnoreQuotes dissapeared

### DISCUSSION THREAD

**Discussion by DocGitHub** [JoshClose/CsvHelper#1697] [Q&A]
**Describe the bug**
The IgnoreQuotes function has dissapeared at least in VB.NET since the commit below:

https://github.com/JoshClose/CsvHelper/commit/3b14b70fd1e9ce742375fbb116799b19fd0e7ccd#diff-95e0818646ed90efcbdd63bdb0f11a016847f10047e7d221e8abd22941d9ec31

For example 'IgnoreBlankLines ' has been made readonly, but IgnoreQuotes  ?? Gone.

Please update documentation or revive IgnoreQuotes

**Comment by JoshClose** [comment]
There are modes now which should make the feature more clear.

https://github.com/JoshClose/CsvHelper/blob/master/src/CsvHelper/CsvMode.cs

When using VB.NET, you'll need to use the `CsvConfiguration` constructor parameters to set values. VB.NET doesn't have support for `init` properties unfortunately. 

You'll want `CsvMode.Escape`, since it ignores quotes.
> Uses escapes. If a field contains a CsvConfiguration.Delimiter, CsvConfiguration.NewLine, or CsvConfiguration.Escape, it is preceded by CsvConfiguration.Escape.
